packages/muelu/test/unit_tests/Regression.cpp

```diff
@@ -103,17 +105,26 @@ TEUCHOS_UNIT_TEST_TEMPLATE_4_DECL(Regression, H2D, Scalar, LocalOrdinal, GlobalO
   // confirm that we did get a hierarchy with two levels -- a sanity check for this test
   TEST_EQUALITY(2, H->GetGlobalNumLevels());
 
+  // When Kokkos Kernels uses TPLs, some Kokkos::deep_copy in the Kokkos Kernels native implementations are not called.
+#if  defined(KOKKOSKERNELS_ENABLE_TPL_CUSPARSE) \
+  || defined(KOKKOSKERNELS_ENABLE_TPL_ROCSPARSE) \
+  || defined(KOKKOSKERNELS_ENABLE_TPL_MKL)
+  constexpr int kkNativeDeepCopies = 0;
```

Comment:
This probably needs to be change into something like
```
#if  defined(KOKKOSKERNELS_ENABLE_TPL_CUSPARSE)
if (Tpetra node is Cuda)
```
and so on. In general this rarely matters, but some of the nightlies might be built with Cuda+Serial enabled.